### DISCUSSION TITLE
SIMD operations

### DISCUSSION THREAD

**Discussion by PanSzelescik** [PurpurMC/Purpur#1492] [__Completed__] [answered]
SIMD operations shouldn't be also enabled on Java 20 and 21 (and in future versions)?
https://github.com/PurpurMC/Purpur/blob/ec60d4826eee00a1df37d0d4a98b05b84f3084d3/patches/server/0001-Pufferfish-Server-Changes.patch#L720

**Comment by YouHaveTrouble** [comment]
It's a pufferfish patch. if you wish for this to change, then go bother Kevin about it.
Personally I haven't made any investigations if this patch will work on versions of java other than 17 so I'm leaving the decision to Kevin as he authored the patch and probably knows better if it's safe to enable past java 17

**Comment by Potothingi** [comment]
Kevin is very lazy so adding this to '0003-Fix-pufferfish-issues.patch' would be great.

https://github.com/pufferfish-gg/Pufferfish/issues/89
https://github.com/pufferfish-gg/Pufferfish/issues/81

**Comment by granny** [comment, marked as answer]
This was added as of build # 2163.
